Doc/library/gettext.rst

```diff
@@ -379,8 +418,37 @@ unexpected, or if other problems occur while reading the file, instantiating a
              n) % {'num': n}
 
 
+   .. method:: pgettext(context, message)
+
+      Look up the *context* and *message* id in the catalog and return the
+      corresponding message string, as an 8-bit string encoded with the
```

Comment:
This is not true in Python 3. The result is a Unicode string, not an encoded 8-bit string. Only `l*gettext` functions return bytes, and this makes them virtually useless.